lib/msf/core/payload/python/meterpreter_loader.rb

```diff
@@ -142,6 +144,214 @@ def stage_meterpreter(opts={})
     met
   end
 
+  def python_encryptor_loader
+    aes_encryptor = Rex::Text.encode_base64(python_aes_source)
+    rsa_encryptor = Rex::Text.encode_base64(python_rsa_source)
+    %Q?
+import codecs,imp,base64
+met_aes = imp.new_module('met_aes')
+met_rsa = imp.new_module('met_rsa')
+exec(compile(base64.b64decode(codecs.getencoder('utf-8')('#{aes_encryptor}')[0]),'<string>','exec'), met_aes.__dict__)
```

Comment:
@smcintyre-r7 thanks for the comments!

1. I hadn't, but that's definitely something to look at for sure.
1. That's fair. I have no problem with that. I guess the reason I didn't do it up front is moreso because the payloads repo was intended to remove those artifacts from MSF and have them inside gems, and I felt like that adding python to the MSF on disk wasn't "the right thing to do". This was a bit of a mid-way cheat :) I definitely think that keeping it out of the body of meterpreter for dev purposes is the way to go. Happy to move it to a template on disk, but it'd mean having two (one for RSA/DER, one for AES).